WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryViewModel.cs

```diff
@@ -185,24 +186,31 @@ public HistoryViewModel(WalletViewModel walletViewModel, IObservable<Unit> updat
 
 	public void SelectTransaction(uint256 txid)
 	{
-		var txnItem = Transactions.FirstOrDefault(item =>
+		var node = TransactionItemNode.Create(Transactions)
+			.SelectMany(node => node.Children.Concat(new [] { node }))
+			.FirstOrDefault(x => x.Item.Id == txid);
+
+		if (node == null)
 		{
-			if (item is CoinJoinsHistoryItemViewModel cjGroup)
-			{
-				return cjGroup.CoinJoinTransactions.Any(x => x.TransactionId == txid);
-			}
+			return;
+		}
+		
+		if (node.Parent != null)
+		{
+			node.Parent.Item.IsExpanded = true;
+		}
 
-			return item.Id == txid;
-		});
+		var indexPath = node.Parent != null ? new IndexPath(node.Parent.Index, node.Index) : new IndexPath(node.Index);
```

Comment:
I have found when I sort table by other column (eg. Balance) the scrolling to selected item does not work.